x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/links/link_to_uptime.tsx

```diff
@@ -14,7 +14,7 @@ import { useKibanaContextForPlugin } from '../../../../../../hooks/use_kibana';
 import type { InventoryItemType } from '../../../../../../../common/inventory_models/types';
 import type { HostNodeRow } from '../../../hooks/use_hosts_table';
 
-interface LinkTUptimeProps {
+export interface LinkTUptimeProps {
```

Comment:
question: Link**T**UptimeProps is a typo or is it supposed to be like that?

Reply:
Thanks for spotting that, it was an old typo, fixed it ✔️